build-showmewebcam.sh

```diff
@@ -9,35 +11,36 @@ if [ ! -d "$BUILDROOT_DIR" ]; then
 fi
 
 export BOARDNAME=$1
+shift
 
 case "$BOARDNAME" in
   raspberrypi0)
-  ;;
-  raspberrypi0w)
-  ;;
+    ;;
+  raspberrypi0_2)
+    ;;
   raspberrypi4)
-  ;;
+    ;;
   *)
-    echo "usage: BUILDROOT_DIR=buildroot $0 (boardname)"
-    echo "boardname: raspberrypi0, raspberrypi0w, raspberrypi4"
+    echo "usage: BUILDROOT_DIR=buildroot $0 (boardname) [component]"
+    echo "boardname: raspberrypi0, raspberry0_2, raspberrypi4"
     exit 1
   ;;
 esac
 
+# Add overridden buildroot packages
+cp -R package-override/* "$BUILDROOT_DIR/package"
```

Comment:
See above, we should definitely stay away from package overrides and just wait for the next stable of buildroot